HISTORY.md

```diff
@@ -1,3 +1,20 @@
+## [3.3.0](https://github.com/theforeman/puppet-pulpcore/tree/3.3.0) (2021-04-21)
```

Comment:
the update here is to prevent changelog-generator from wanting to insert empty release notes for v2.2.3